tests/x11regressions/libreoffice/libreoffice_double_click_file.pm

```diff
@@ -34,7 +35,12 @@ sub run {
         send_key_until_needlematch("libreoffice-specified-list-$tag", "right", 50, 1);
         assert_and_dclick("libreoffice-specified-list-$tag");
         assert_screen("libreoffice-test-$tag", 90);
-        send_key 'ctrl-q';
+        if (sle_version_at_least('15')) {
```

Comment:
Same as above for `is_sle`:)

Reply:
Thanks, I just fixed it, please help to review.